web/moduly/prihlaska/prihlaska.php

```diff
@@ -262,9 +262,16 @@ function nahledPredmetu(string $cestaKObrazku): string
     'zaplatitNejpozdejiDo'            => $systemoveNastaveni->nejpozdejiZaplatitDo()->format(DateTimeCz::FORMAT_DATUM_LETOS),
 ]);
 
-$t->parse($u->gcPrihlasen()
-    ? 'prihlaska.prihlasen'
-    : 'prihlaska.neprihlasen');
-if ($u->gcPrihlasen() && pred($systemoveNastaveni->gcBeziOd())) {
-    $t->parse('prihlaska.odhlasit');
+if ($u->gcPrihlasen()) {
+    if ($u->vekKDatu($systemoveNastaveni->gcBeziOd()) < 15 &&
+        (!$u->potvrzeniZakonnehoZastupceOd() ||
+            $u->potvrzeniZakonnehoZastupceOd()->format('y') !== date('y'))) {
```

Comment:
Logicky správně a v běžném provozu tohle bude OK.

Gamecon ale žije v jakmsi "účetním roce", nikoli kalendářním. Říkáme tomu prostě ročník, mění se až někdy koncem ledna tzv. "překlopením ročníku" (po něčem jako účetní uzávěrka).

To by mohlo dělat potíže při trestech nebo v zálohách starších ročníků (až zamrazíme 2024 podobně jako https://2023.gamecon.cz/)

Použij tu `$systemoveNastaveni->rocnik()`